e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -500,14 +508,14 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
 
       saveDashboard();
 
-      cy.findByTestId("field-set").click();
+      cy.button(DASHBOARD_FILTER_TEXT.name).click();
```

Comment:
Not sure why but `cy.findByTestId("field-set").click()` crashed Cypress for me a lot.
I does not happen with `cy.button(DASHBOARD_FILTER_TEXT.name).click()`.